### DISCUSSION TITLE
Home All Function

### DISCUSSION THREAD

**Discussion by rthuser** [Jyers/Marlin#1930] [General Discussion]
Hello, great work on new and upgrades.
Using firmware E3V2-BLTouch-3x3-v4.2.2-v2.0.1.bin i have a question.
What should happen when i use "Home All" in prepare --> homing?
Only Z axis is homing. Using "x home" and "y home" separately works fine.
Set "home position" is answered with a beep but "Home All" is not working.

**Comment by Cal-LF** [comment]
@rthuser You can download the 2.0.2h release of precompiled .bin files (https://github.com/Jyers/Marlin/files/7808537/E3V2-Pre-compiled-firmwares-v2.0.2h.zip)
Try the one for your configuration. It may resolve your homing issue. The zip is from here. #1694

For the latest upgrade to this firmware go here. https://github.com/tititopher68-dev2/Marlin/tree/new_feature
You will have to compile your own from the source files.

**Comment by rthuser** [comment]
Thank you very much. Using this firmware unfortunately no change.

Is there a manual somewhere in which each menu item is explained?
